conf/application.conf

```diff
@@ -67,6 +67,10 @@ microservice {
         host = localhost
         port = 9250
       }
+      vat-api {
```

Comment:
Is there an app-config PR for this too?

Reply:
not yet, literally just made this change